### PR TITLE
Fixes gh-5: Adds updated usage documentation to the README.

### PR DESCRIPTION
**Pull request by colinbdclark**
@the-t-in-rtf, how does this look? I'll publish a dev release once you give me the thumbs up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,17 @@ Before you can use this package, you will need to install its dependencies using
...
+Alternatively, if the router is installed as a local dependency of your project, you can run:
 
-You can also install flocking-midi-router as a global command available in your path by running `npm link` (or `yarn
-link`).  This will add a command named `flocking-midi-router` to your path.  If you launch either variation without any
-arguments (or with the `--help` option), usage information will be displayed.
+`node node_modules/.bin/flocking-midi-router`
+
+Or, if you have cloned the repository and want to run it directly, use:
+
+`node node_modules/launcher.js`
```
Comment (the-t-in-rtf) [inline README.md]: This should be `node launcher.js`

**Comment by the-t-in-rtf** [comment]
I found one typo in the instructions and commented on the appropriate line.

I tested the "run when installed as a dependency" route with a git:// + commit hash reference, it's just fine.

We won't be able to test the global install until it's published at least in dev form.  If you're OK with having a dev release out there briefly, you can publish a dev release, then we can verify before merging and publishing the initial release.  Also happy to merge once the typo is fixed and just agree to immediately verify and create a new ticket if there are any problems.

**Comment by colinbdclark** [comment]
I must be going crazy, @the-t-in-rtf, but I can't find your comment where you point out the typo in the instructions!

**Comment by the-t-in-rtf** [comment]
Sorry @colinbdclark, the new GitHub interface tricked me.  Instead of adding a comment, it had created a draft review that was waiting to be saved.  It's there now.

**Pull request merged**